src/main/java/org/apache/tomcat/jni/SSL.java

```diff
@@ -234,6 +234,9 @@
     public static final int SSL_SELECTOR_FAILURE_NO_ADVERTISE = 0;
     public static final int SSL_SELECTOR_FAILURE_CHOOSE_MY_LAST_PROTOCOL = 1;
 
+    public static final int SSL_ST_CONNECT = 0x1000;
```

Comment:
Ok that is fine for this PR....but should we create another issue to expose defines from C to java? Seems like it would be an improvement to use the "real" definitions.